src/shared_utilities/DataHandling.jl

```diff
@@ -0,0 +1,246 @@
+module DataHandling
+
+using Dates
+
+import ClimaCore
+
+import ..FileReader: NCFileReader
+import ..Regridder
+import ..Regridder:
+    AbstractRegridder, TempestRegridder, regrid, AVAILABLE_REGRIDDERS
```

Comment:
I dont see anything in the style guide for all caps in names - why is AVAILABLE_REGRIDDERS capitalized?

Reply:
ah - I see some languages use that for constants. it doesnt seem like Julia follows that notation: https://docs.julialang.org/en/v1/base/base/#const

Reply:
ALL_CAPS is mostly a way to tell the reader that the variable is a "global fixed parameter", with the emphasis being in "global". We don't have to do this, but I personally find it an useful additional qualifier.

(`const` in Julia doesn't even ensure that the value doesn't change.)